ext/oauth-client/Civi/Api4/OAuthSessionToken.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Civi\Api4;
+
+use Civi\Api4\Generic\AbstractAction;
+use Civi\Api4\Generic\Result;
+
```

Comment:
```suggestion

/**
 * OAuth Access Tokens.
 *
 * @see https://docs.civicrm.org/dev/en/latest/framework/oauth/#model-token
 *
 * @primaryKey cardinal
 * @searchable none
 * @since 5.67
 * @package Civi\Api4
 */
```